### PR TITLE
Automatically add $SUBDOMAIN to settings.

### PR DESCRIPTION
**Pull request by enoch85**
This makes it fully automatic

### CODE REVIEW

**Comment by enoch85** [comment]
@Ark74 Have you tested it? Does it work?

**Comment by enoch85** [comment]
Tested and works! Thanks @nickvergessen!

File: static/collabora.sh [hunk 1]
```diff
@@ -274,9 +274,9 @@ fi
 if [ -d $NCPATH/apps/richdocuments ]
 then
     sudo -u www-data php $NCPATH/occ app:enable richdocuments
+    sudo -u www-data $NCPATH/occ config:app:set richdocuments wopi_url --value="$SUBDOMAIN"
```
Comment (nickvergessen) [inline static/collabora.sh]: Missing https?

Comment (enoch85) [inline static/collabora.sh]: Aah great catch! Thanks!

**Pull request merged**